src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -43,12 +42,13 @@
     /**
      * Analyzes the lines of the file, given in the {@code fileInfo}, that has changed in the time period provided
      * by {@code config}.
-     * Returns null if the file contains the reused tag, the file is missing from the local system, or none of the
+     * Returns null if the file is missing from the local system, or none of the
      * {@code Author} specified in {@code config} contributed to the file in {@code fileInfo}.
      */
     public static FileResult analyzeFile(RepoConfiguration config, FileInfo fileInfo) {
         String relativePath = fileInfo.getPath();
-        if (isReused(config.getRepoRoot(), relativePath)) {
+
+        if (isEmptyFile(config.getRepoRoot(), relativePath)) {
```

Comment:
This `isReused` method was originally checking 2 things:
(1) if the file is empty
(2) if the first line of the file contains the `//@reused` string

We would still want to keep the first check. However, shouldn't we check **whether the file exists** before checking if the file is empty? Currently, the `analyzeFile` method checks if the file is empty **before** confirming that the file exists.

Reply:
I believe it should be checked somewhere in the previous step, but you can recheck.

Reply:
Done - I've placed the check for file existence before checking that the file is non-empty.